cpp/include/raft/core/cusparse_macros.hpp

```diff
@@ -59,8 +60,8 @@ namespace detail {
 
 inline const char* cusparse_error_to_string(cusparseStatus_t err)
 {
-#if defined(CUDART_VERSION) && CUDART_VERSION >= 10100
-  return cusparseGetErrorString(err);
+#if defined(CUDART_VERSION) && CUDART_VERSION >= 10010
+  return cusparseGetErrorString(err)
```

Comment:
```suggestion
  return cusparseGetErrorString(err);
```
Is there a reason for this `;` to disappear?

Reply:
Nope, I just saw that, sorry again